textAdventure.js

```diff
@@ -0,0 +1,229 @@
+            pickup('torch')
+            temple.description= 'a cold stone temple.  there is one torch left'
+    }else if(choice==='pick up torch'& temple.items.length===1 && currentRoom.name===temple.name){
+                pickup('torch')
+                temple.description= 'a cold stone temple.'
+    }else if (choice.includes('pick up')){
+            pickup(choice.replace('pick up ',''))
+    }else if(choice ==='use torch' && cave.description==='The cave is very dark.  The growling is much louder'&& currentRoom.name===cave.name && player.inventory.includes(torch)){
+        console.log("light fills the room.  You see a hibernating bear")
+        cave.description='a sleeping bear.  It is very cute'
+    }else if (choice=== 'use holy symbol' &&currentRoom.name==='riverBridge' && player.inventory.includes(holySymbol)){
+            console.log('\nThe troll recoils in disgust\nand stumbles off of the bridge\n"AW DAAAAAaaaaaaannnnnnngggggg....."\nyou hear a splash below')
+            riverBridge.description='a clear bridge lies before you'
+            console.log(riverBridge.description)
+            riverBridge.east='passedBridge'
+            action();
+    }else if(choice ===`use sword` && currentRoom.name===cave.name && player.inventory.includes(sword)){
+            console.log('You slayed a sweet little bear.  How could you.')
+            cave.description = 'Here lies a dead bear. In a cave. Looks like it was sleeping. Weird.'
+            woodsCave.description='a cave. It is silent'
+    }else if(choice==='use sword' && currentRoom.description==='A troll guards the\nbridge to the east;\nover it, a massive bridge.\n"NONE SHALL PASS," it screams'&& player.inventory.includes(sword)){
+        console.log('\nYou run the troll through with your sword\n but the wounds close up quickly\n "HAHAHA," it says, "YOUR MORTAL WEAPONS WILL NOT WORK ON ME"\nit throws you back and resumes its previous stance')
+
+    }else if(choice==='pet bear' && currentRoom.description==='a sleeping bear.  It is very cute'){
+       console.log('the bear begins to stir.  It awakens and looks\ncuriously at you')
+       cave.description='The bear is awake, and looks\ncuriously at you.'
+    }else if(choice==='pet bear'&& currentRoom.description==='The bear is awake, and looks\ncuriously at you.'){
+        console.log("the bear's hind leg begins to shake uncontrollably.\nhe gives you a big slobbery kiss")
+        cave.description='a happy bear :)'
+    }else if(choice==='pet bear'&& currentRoom.name==cave.name&&currentRoom.description!=='The cave is very dark.  The growling is much louder'){
+        console.log("the bear licks your hand as you pet it")
+        cave.description='a happy bear :)'
+    }else if(choice==='read sign' && currentRoom.name===meadow2.name){
+        console.log('The sign reads:\n <----Bear Cave  Grandmas Pie Palace---->')
+    }else if (choice==='enter' || 'open door' &&currentRoom.name===grandmasHouse.name){
```

Comment:
remember to do the comparison check on both sides of the logical operators.  As this is 'open door' will always be truthy as it's a string. To make it a valid comparison it should be `choice === 'enter' || choice === 'open door'`